src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,53 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class LoginPage extends BasePage {
+    public static LoginPage loginPage(){
+        return new LoginPage();
+    }
+        // locators
+        @FindBy(css= ("[type='password']"))
+        WebElement passwordInput;
+        @FindBy(xpath=("//input[@type='email']"))
+        WebElement emailInput;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```